lib/device.js

```diff
@@ -251,6 +256,174 @@ const util = require('util'),
                 }
                 return returnTxt;
             }
+        },
+        /**
+         * get screen capture of the given device
+         * @property options {String} device, display, inputPath
+         */
+        captureScreen: function(options, next) {
+            log.info("device#capture#captureScreen()");
+            if (typeof next !== 'function') {
+                throw new Error('Missing completion callback (next=' + util.inspect(next) + ')');
+            }
+            options = options || {};
+            async.series([
+                _makeSession,
+                _makeCaptureOption,
+                _captureFile,
+                _copyToLocal
+            ],  function(err, results) {
+                log.verbose("device#capture()", "err: ", err, "results:", results);
+
+                let resultTxt = "";
+                resultTxt = "Create " + chalk.green(options.capFilename) + " to " + options.destinationPath +"\nSuccess";
+
+                // clean up /tmp/capture directory in target device
+                if (options.targetCapDir) {
+                    _removeCaptureDir(function finish (removeErr){
+                        next(err ? err : removeErr, resultTxt);
+                    });
+                } else {
+                    next(err, resultTxt);
+                }
+            });
+
+            function _makeSession(next) {
+                options.nReplies = 1; // -n 1
+                options.session = new novacom.Session(options.device, next);
+            }
+
+            function _makeCaptureOption(next) {
+                log.info("device#capture#_makeCaptureOption()");
+                let capFilename = "";
+                let capFormat = "PNG"; // PNG is default format
+                let outputDirPath;
```

Comment:
let capFilename = "",
capFormat = "PNG",
outputDirPath;

is better...I think.

Reply:
Done.